examples/get_tree.cpp

```diff
@@ -185,6 +185,14 @@ void print_i3_tree(const i3_containers::node& a_node, const std::string& a_inden
     }
     std::cout << a_indent << "Fullscreen mode: " << fullscreen_mode << "\n";
 
+    // Print marks
+    std::cout << a_indent << "Marks: [";
+    for (auto const& mark : a_node.marks)
```

Comment:
Lets just west `const` to be consistent.
```suggestion
    for (const auto& mark : a_node.marks)
```